readme.md

```diff
@@ -80,6 +100,20 @@ $ npm -v
 
 * **Your Answer:**
 
+```
+> echo "Error: no test specified" && exit 1
+Error: no test specified
+npm ERR! Test failed. 
+```
+
+`package.json` contains no pointers to tests:
+
+```
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+```
```

Comment:
This doesn't quite explain what's happening. Just to be clear, when you run `npm test`, it runs the matching value in the script. In this case, the matching value tells the computer to print a message to the screen and exit.